applications/GeoMechanicsApplication/tests/geo_truss_with_reset_displacement/ProjectParameters_stage1.json

```diff
@@ -13,7 +13,7 @@
         "domain_size":                        2,
         "model_import_settings":              {
             "input_type":       "mdpa",
-            "input_filename":   "geo_truss_with_reset_displacement_stage_1"
+            "input_filename":   "geo_truss_with_reset_displacement"
```

Comment:
This name and the directory name of the test are now misleading. We are using the truss element from structural. That was the intention of the whole exercise.

Reply:
Good catch, changed